pkg/controller/schedule_controller.go

```diff
@@ -33,7 +33,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	api "github.com/heptio/velero/pkg/apis/velero/v1"
```

Comment:
@skriss looks like this used to have a duplicate package import too, should I update this to use velerov1api everywhere too?

Reply:
Eh, let's get this one merged - we can update this next time we touch this file.